src/wazuh_modules/vulnerability_detector/wm_vuln_detector_nvd.c

```diff
@@ -4154,4 +4154,20 @@ char *wm_vuldet_generate_msu_request(cpe *s_cpe, vu_feed dist_ver) {
     return request;
 }
 
+void wm_vuldet_normalize_architecture_NVD(char* architecture) {
+    // Correcting the architecture string to match the NVD
```

Comment:
I would say `translating` or `fixing` instead of `correcting`.

Reply:
Fixed!